control/grpc.py

```diff
@@ -1187,6 +1199,12 @@ def get_image_identification(self, rbd_pool: str, rbd_image: str) -> list[ImageI
     def set_image_identification(self, rbd_pool: str, rbd_image: str, img_id: ImageIdentification):
         assert img_id, "Can't set an empty image id"
 
+        if self.fsid is None:
+            self.fsid = self.ceph_utils.fetch_ceph_fsid()
+            self.logger.debug(f"Cluster FSID is {self.fsid}")
+        if img_id.fsid is None:
+            self.logger.debug(f"Will use FSID {self.fsid} for image id {img_id}")
```

Comment:
maybe this will give more context? wdyt?
```suggestion
            self.logger.debug(f"Couldn't find img fsid. Will use FSID {self.fsid} for image id {img_id}")
```

Reply:
will rephrase debug message